docs/tutorials/03_quantum_kernel.ipynb

```diff
@@ -420,13 +567,61 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [
+    {
```

Comment:
We should find a title for this final section....

Reply:
Sure, will look into this